src/lib.rs

```diff
@@ -170,18 +170,22 @@ impl<M: Message> MavFrame<M> {
     pub fn deser(version: MavlinkVersion, input: &[u8]) -> Result<Self, ParserError> {
         let mut buf = Bytes::new(input);
 
+        let sequence = buf.get_u8();
         let system_id = buf.get_u8();
         let component_id = buf.get_u8();
-        let sequence = buf.get_u8();
         let header = MavHeader {
             system_id,
             component_id,
             sequence,
         };
 
         let msg_id = match version {
-            MavlinkVersion::V2 => buf.get_u32_le(),
-            MavlinkVersion::V1 => buf.get_u8().into(),
+            MavlinkVersion::V2 => {
+                let mut bytes = [0u8; 4];
+                bytes[..3].copy_from_slice(buf.get_bytes(3));
+                u32::from_le_bytes(bytes)
+            }
```

Comment:
Would it not make more sense to add a `get_u24_le` method to our bytes implementation instead of handling it in a special way here.

Reply:
Something like the following should work:
```rust
    pub fn get_u24_le(&mut self) -> u32 {
	const SIZE: usize = 3;
	self.check_remaining(SIZE);

	let mut val = [0u8; SIZE + 1];
	val[..3].copy_from_slice(&self.data[self.pos..self.pos + SIZE]);	
        self.pos += SIZE;
        u32::from_le_bytes(val)
    }

    pub fn get_i24_le(&mut self) -> i32 {
	const SIZE: usize = 3;
	self.check_remaining(SIZE);

	let mut val = [0u8; SIZE + 1];
	val[..3].copy_from_slice(&self.data[self.pos..self.pos + SIZE]);	
        self.pos += SIZE;
        i32::from_le_bytes(val)
    }
```

Reply:
This have been implemented in #159 so you can use that